src/cpp/hu.bme.mit.inf.modes3.ESP32-sensor/railsensor_3/ESP32-sensor/lib/RealTimeLength/RealTimeLength.cpp

```diff
@@ -0,0 +1,51 @@
+#include "RealTimeLength.h"
+#include <iostream>
+using namespace std;
+
+RealTimeLength::RealTimeLength(int* Asindex, double* Astack, int* Bsindex, double* Bstack){

```

Comment:
Consider removing every pointer from the code.
First of all, the indices could not pointers. Also Asinxes is wrongly named, it's too easy to read it `AsIndex`.